cfme/tests/cli/test_appliance_update.py

```diff
@@ -85,17 +95,236 @@ def appliance_preupdate(old_version, appliance):
     sp.destroy_pool(pool_id)
 
 
-@pytest.mark.uncollectif(lambda: not store.current_appliance.is_downstream)
-def test_update_yum(appliance_preupdate, appliance):
-    """Tests appliance update between versions"""
+@pytest.fixture(scope="function", )
```

Comment:
**REQUIRED**
Unneeded comma or you can even remove `(scope="function", )` at all.